plugins/obs-ffmpeg/obs-ffmpeg-output.h

```diff
@@ -74,5 +79,34 @@ struct ffmpeg_data {
 	char *last_error;
 };
 
+struct ffmpeg_output {
+	obs_output_t *output;
+	volatile bool active;
+	struct ffmpeg_data ff_data;
+
+	bool connecting;
+	pthread_t start_thread;
+
+	uint64_t total_bytes;
+
+	uint64_t audio_start_ts;
+	uint64_t video_start_ts;
+	uint64_t stop_ts;
+	volatile bool stopping;
+
+	bool write_thread_active;
+	pthread_mutex_t write_mutex;
+	pthread_t write_thread;
+	os_sem_t *write_sem;
+	os_event_t *stop_event;
+
+	DARRAY(AVPacket *) packets;
+#ifdef NEW_MPEGTS_OUTPUT
+	/* used for srt & rist */
+	URLContext *h;
+	AVIOContext *s;
```

Comment:
Could these perhaps be friendlier named variables? `h` and `s` don't give much context when looking at the code.

Reply:
although the plan is eventually to add more and more of our own code, I've kept on purpose the current code very close to avformat/libsrt.c .
This will allow us to port more easily changes/bugfixes done in avformat.
At some point when the divergence will be too large, yeah I agree it'd make sense to have more friendly names.